scripts/build/cc/100-gcc.sh

```diff
@@ -966,38 +1096,22 @@ do_gcc_backend() {
     find . -type f -name "*.la" -exec rm {} \; |CT_DoLog ALL
     CT_Popd
 
-    if [ "${build_manuals}" = "yes" ]; then
-        CT_DoLog EXTRA "Building the GCC manuals"
-        CT_DoExecLog ALL ${make} pdf html
-        CT_DoLog EXTRA "Installing the GCC manuals"
-        CT_DoExecLog ALL ${make} install-{pdf,html}-gcc
-    fi
-
     # Create a symlink ${CT_TARGET}-cc to ${CT_TARGET}-gcc to always be able
     # to call the C compiler with the same, somewhat canonical name.
     # check whether compiler has an extension
     file="$( ls -1 "${CT_PREFIX_DIR}/bin/${CT_TARGET}-gcc."* 2>/dev/null || true )"
     [ -z "${file}" ] || ext=".${file##*.}"
     if [ -f "${CT_PREFIX_DIR}/bin/${CT_TARGET}-gcc${ext}" ]; then
-        CT_DoExecLog ALL ln -sfv "${CT_TARGET}-gcc${ext}" "${CT_PREFIX_DIR}/bin/${CT_TARGET}-cc${ext}"
+        CT_DoExecLog ALL ln -sfv "${CT_TARGET}-gcc${ext}" "${prefix}/bin/${CT_TARGET}-cc${ext}"
     fi
 
-    if [ "${CT_CANADIAN}" = "y" ]; then
-        CT_DoLog EXTRA "Canadian Cross unable to confirm multilibs configured correctly"
-    else
-        multilibs=( $( "${prefix}/bin/${CT_TARGET}-gcc" -print-multi-lib \
-                       |tail -n +1 ) )
-        if [ ${#multilibs[@]} -ne 0 ]; then
-            CT_DoLog EXTRA "gcc configured with these multilibs (including the default):"
-            for i in "${multilibs[@]}"; do
-                dir="${i%%;*}"
-                flags="${i#*;}"
-                flags=${flags//@/ -}
-                osdir=$( "${prefix}/bin/${CT_TARGET}-gcc" -print-multi-os-directory $flags )
-                CT_DoLog EXTRA "   '${flags}' -->  lib/${dir}/ (gcc)   lib/${osdir} (os)"
-            done
-        else
-            CT_DoLog WARN "no multilib configuration: GCC unusable?"
-        fi
+    cc_gcc_multilib_housekeeping cc="${prefix}/bin/${CT_TARGET}-gcc" \
+        host="${host}"
+
+    if [ "${build_manuals}" = "yes" ]; then
+        CT_DoLog EXTRA "Building the GCC manuals"
+        CT_DoExecLog ALL ${make} pdf html
+        CT_DoLog EXTRA "Installing the GCC manuals"
+        CT_DoExecLog ALL ${make} install-{pdf,html}-gcc
```

Comment:
Four months after making the change, I don't remember the exact reason for this. I think it was just separating docs build from the actual compiler build & tweaking. I don't think it will cause any churn - it is going to do the same stuff, just in different order. But I'll restore the previous order.